python/nav/web/watchdog/views.py

```diff
@@ -26,58 +26,107 @@
 
 def render_index(request):
     """Controller for WatchDog index"""
-    navpath = [('Home', '/'), ('WatchDog', )]
+    navpath = [("Home", "/"), ("WatchDog",)]
 
     context = {
-        'navpath': navpath,
-        'title': create_title(navpath),
-        'tests': get_statuses()
+        "navpath": navpath,
+        "title": create_title(navpath),
+        "tests": get_statuses(),
     }
 
-    return render(request, 'watchdog/base.html', context)
+    return render(request, "watchdog/base.html", context)
 
 
 def get_active_addresses(_):
     """Get active addresses on the network"""
     active = Arp.objects.filter(end_time=INFINITY)
     num_active = active.count()
-    num_active_ipv6 = active.extra(where=['family(ip)=6']).count()
-    num_active_ipv4 = active.extra(where=['family(ip)=4']).count()
-    return JsonResponse({
-        'active': num_active,
-        'ipv6': num_active_ipv6,
-        'ipv4': num_active_ipv4
-    })
+    num_active_ipv6 = active.extra(where=["family(ip)=6"]).count()
+    num_active_ipv4 = active.extra(where=["family(ip)=4"]).count()
+    return JsonResponse(
```

Comment:
Blackified? More readable before.

Reply:
of course the wrong line was selected.. meant the formatting of the dict in `JsonResponse`.

Reply:
Agreed, but Black decided otherwise. See 998f0b7